library/std/src/os/fd/raw.rs

```diff
@@ -84,7 +84,10 @@ pub trait FromRawFd {
     ///
     /// # Safety
     ///
-    /// The `fd` passed in must be a valid and open file descriptor.
+    /// The `fd` passed in must be an [owned file descriptor][io-safety];
+    /// in particular, it must be valid and open.
```

Comment:
It's on my todo list to remove the "valid and" here, as it's ambiguous and redundant at best.

Reply:
I removed it.